scripts/clean_results_csv.py

```diff
@@ -36,29 +47,46 @@ def main():
 
     ## this is a bunch of csv ingesting that is very dependent on the way the csv looks
     dr.df = dr.df.drop("Unnamed: 0", axis=1)
-    dr.df["MERS_structure"] = dr.df["MERS_structure"].apply(
-        lambda x: x.split("_")[1]
-    )
+    if not args.keep_structure_source:
+        dr.df["MERS_structure"] = dr.df["MERS_structure"].apply(
+            lambda x: x.split("_")[1]
+        )
 
     ## Rename the columns, add POSIT_R
-    dr.df.columns = [
-        "Compound_ID",
-        "Structure_Source",
-        "Docked_File",
-        "RMSD",
-        "POSIT",
-        "Chemgauss4",
-        "Clash",
-    ]
+    print(args.dimer)
+    if args.dimer:
+        dr.df.columns = [
+            "Compound_ID",
+            "Structure_Source",
+            "Dimer",
+            "Docked_File",
+            "RMSD",
+            "POSIT",
+            "Chemgauss4",
+            "Clash",
+        ]
+    else:
+        dr.df.columns = [
+            "Compound_ID",
+            "Structure_Source",
+            "Docked_File",
+            "RMSD",
+            "POSIT",
+            "Chemgauss4",
+            "Clash",
+        ]
     dr.df["POSIT_R"] = 1 - dr.df.POSIT
 
     ## Drop "_bound" from Compound_ID
-    dr.df.Compound_ID = [
-        string.replace("_bound", "") for string in dr.df.Compound_ID
-    ]
+    # ToDo: deal with this issue of the "_bound" compound IDs in the fragalysis database.
+    # dr.df.Compound_ID = [
+    #     string.replace("_bound", "") for string in dr.df.Compound_ID
+    # ]
 
     ## Add Complex_ID
-    dr.df["Complex_ID"] = f"{dr.df.Compound_ID}_{dr.df.Structure_Source}"
+    ## This is not the same thing as f"{dr.df.Compound_ID}_{dr.df.Structure_Source}", as this enables rowwise addition
+    ## as opposed to adding the *entire* series as a single string
```

Comment:
We can just remove this line/suggestion since it is not the same thing. My bad!
```suggestion
```

Reply:
eh lets keep it so that we don't change it again?